### PR TITLE
LPS-185849 Remove multiple configurations in JournalPortlet

### PR DESCRIPTION
**Pull request by julschong**
resummited to fix tests from #4892

### CODE REVIEW

**Comment by julschong** [comment]
@Kvale1733 Hi Kevin,
I could not replicate this regression locally:
`[exec] com.liferay.asset.search.test.AssetSearcherClassNameIdsTest > testAll FAILED
     [exec]     java.lang.AssertionError: [{uid=[com.liferay.bookmarks.model.BookmarksEntry_PORTLET_33663], entryClassPK=[33663], companyId=[20096], entryClassName=[com.liferay.bookmarks.model.BookmarksEntry]}, {uid=[com.liferay.journal.model.JournalArticle_PORTLET_33667], entryClassPK=[33668], companyId=[20096], entryClassName=[com.liferay.journal.model.JournalArticle]}}, length=2, query=null] expected:<3> but was:<2>
     [exec]     	at org.junit.Assert.fail(Assert.java:89)
     [exec]     	at org.junit.Assert.failNotEquals(Assert.java:835)
     [exec]     	at org.junit.Assert.assertEquals(Assert.java:647)
     [exec]     	at com.liferay.asset.search.test.AssetSearcherClassNameIdsTest.testAll(AssetSearcherClassNameIdsTest.java:90)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)`

This is what I get:
`com.liferay.asset.search.test.AssetSearcherClassNameIdsTest > classMethod FAILED
    org.osgi.framework.BundleException: Could not resolve module: com.liferay.asset.test [4473]
      Unresolved requirement: Import-Package: com.liferay.bookmarks.model
        at org.eclipse.osgi.container.Module.start(Module.java:444)
        at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:428)
        at org.eclipse.osgi.internal.framework.EquinoxBundle.start(EquinoxBundle.java:447)
        at com.liferay.arquillian.extension.junit.bridge.connector.FrameworkCommand.lambda$startBundle$37e16ea2$1(FrameworkCommand.java:46)
        at com.liferay.arquillian.extension.junit.bridge.connector.ArquillianConnectorThread.run(ArquillianConnectorThread.java:82)

1 test completed, 1 failed`

**Comment by Kvale1733** [comment]
Hey @julschong. You need to deploy bookmarks-api first since it is no longer deployed by default due to this issue: https://liferay.atlassian.net/browse/LPS-188724, which I investigated more to get a resolution. But that's unrelated to your PR.

This is reproducable if you deploy bookmarks-api and then run the test.

**Comment by julschong** [comment]
@Kvale1733 I cannot reproduce `AssetSearcherClassNameIdsTest` regression locally even after deploying bookmarks-api.
I even tested with all the test groups on ci to make sure. 
I'll push to fix SF and rerun tests to see

**Comment by julschong** [comment]
ci:test:echo

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalDDMTemplateManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -195,6 +197,9 @@ public Boolean isShowCreationMenu() {
 				return true;
 			}
 		}
+		catch (ConfigurationException configurationException) {
+			throw new RuntimeException(configurationException);
```
Comment (tinatian) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalDDMTemplateManagementToolbarDisplayContext.java:201]: @julschong Why it is throwing runtime exception?

**Comment by julschong** [comment]
need to change to still use request to pass configurations so that we don't keep duplicating the configuration

**Pull request closed without merging**